services/meta/service_test.go

```diff
@@ -205,3 +477,75 @@ func mustParseStatement(s string) influxql.Statement {
 	}
 	return stmt
 }
+
+// Test long poll of snapshot.
```

Comment:
can you remove all this commented stuff? The long poll is already tested on every call the client makes that creates something.